math-native/src/main/java/org/ballerinalang/stdlib/math/utils/MathUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.stdlib.math.utils;
+
+import io.ballerina.runtime.api.Module;
+
+import org.ballerinalang.stdlib.math.nativeimpl.ModuleUtils;
+
+public class MathUtils {
+
+    /**
+     * Gets ballerina math package.
+     *
+     * @return math package.
+     */
+    public static Module getMathPackage() {
```

Comment:
Why do we need two separate methods to do the same thing?

Reply:
Removed in 3a2e918